README.md

```diff
@@ -0,0 +1,44 @@
+# Firmware for Signet Devices
+
+## Prerequisites
+
+#### Cloning Directions
+
+This project uses git submodules, you will either need to use `git clone
+--recursive <repourl>` or after cloning use `git submodule update --init`.
+
+#### Dependencies
+
+The following packages must be installed:
+
+`apt install build-essential autoconf automake gperf texinfo help2man ncurses-dev bison flex libtool-bin libjson-c-dev libiberty-dev`
+
+## Building Firmwares
+
+#### Prepare the Toolchain, Dependencies and Tools
+
+For the legacy Signet go to `firmware` directory, for Signet High-capacity go
+to `firmware-hc`, and run once `./build-deps.sh` to build the corresponding
+toolchain and dependencies.
+
+For packing the firmware in a suitable format for the Signet client GUI we will
+need a firmware encoder too. `make signet-firmware-encoder` for Signet and
+`make hc-firmware-encoder` for Signet High-capacity.
+
+#### Build the Firmware
+
```

Comment:
The steps for Signet legacy vs Signet HC are different enough that I think it justifies having two sections such as "Building the Signet legacy firmware" and "Building the Sigent HC firmware"

Reply:
Yes, I was also considering that. Done. Please see changes.

But eventually, it would be easier, IMO, to have an identical process for both:
1) ./build-deps.sh
2) make
3) optionally, ./firmware-encoder ...

Not a priority :)